src/Control/Parallel.purs

```diff
@@ -16,6 +18,16 @@ import Control.Parallel.Class (class Parallel, parallel, sequential, ParCont(..)
 import Data.Foldable (class Foldable, traverse_, oneOfMap)
 import Data.Traversable (class Traversable, traverse)
 
+parApply
+  :: forall f m a b
+   . Parallel f m
+   => m (a -> b)
+   -> m a
+   -> m b
+parApply mf ma = apply (parallel mf) (parallel ma) # sequential
+
+infixl 4 apply as <!>
```

Comment:
This choice of name seems a bit arbitrary. Now, I'll admit, it's no more arbitrary than `<*>` or the recently added `<@>`, but I wonder if we need it at all. I might even just use `parApply` as an infix function if I need this.

Reply:
I would rather not claim this name in a core library for something which the user can alias themselves.